packages/office-ui-fabric-react/src/components/KeytipLayer/KeytipLayer.base.tsx

```diff
@@ -513,6 +515,18 @@ export class KeytipLayerBase extends React.Component<IKeytipLayerProps, IKeytipL
       this._delayedKeytipQueue = this._delayedKeytipQueue.concat(this._keytipTree.currentKeytip?.children || []);
       this._addKeytipToQueue(sequencesToID(keytipProps.keySequences));
     }
+
+    if (this._newCurrentKeytipSequences && arraysEqual(keytipProps.keySequences, this._newCurrentKeytipSequences)) {
+      this._triggerKeytipImmediately(keytipProps);
+    }
+
+    if (this._isCurrentKeytipAnAlias(keytipProps)) {
+      let keytipSequence = keytipProps.keySequences;
+      if (keytipProps.overflowSetSequence) {
+        keytipSequence = mergeOverflows(keytipSequence, keytipProps.overflowSetSequence);
+      }
+      this._keytipTree.currentKeytip = this._keytipTree.getNode(sequencesToID(keytipSequence));
+    }
```

Comment:
can we extract this code (also used in `_onKeytipAdded`) to avoid duplication?

Reply:
yeah sure I'll do that refactor on both branches